CMakeLists.txt

```diff
@@ -84,11 +84,16 @@ if(WIN32)
         SET(CMAKE_INSTALL_PREFIX "${CMAKE_BINARY_DIR}/Install" CACHE PATH "Cmake install path" FORCE)
     endif()
 
-    # might work with "SHARED" as well.
-    set(PA_LIBRARY_TYPE "STATIC")
+    # PortAudio: use static library
+    set(PA_BUILD_STATIC ON CACHE BOOL "PortAudio static library")
+    set(PA_BUILD_SHARED OFF CACHE BOOL "PortAudio shared library")
+    # Call static library "portaudio_static" since this will also cause the inclusion of the required architecture suffix
+    set(PA_LIBNAME_ADD_SUFFIX ON CACHE BOOL "PortAudio static library suffix")
+    # disable PortAudio install target
+    set(PA_DISABLE_INSTALL ON CACHE BOOL "Disable PortAudio install")
     if(NOT MSYS)
-      set(PORTAUDIO_INCLUDE_DIRS "${CMAKE_SOURCE_DIR}/external_libraries/portaudio_sc_org/include" CACHE PATH "Portaudio include directory" FORCE)
-      add_subdirectory("${CMAKE_SOURCE_DIR}/external_libraries/portaudio_sc_org")
+      set(PORTAUDIO_INCLUDE_DIRS "${CMAKE_SOURCE_DIR}/external_libraries/portaudio/include" CACHE PATH "Portaudio include directory" FORCE)
```

Comment:
do you happen to know if this line is strictly necessary? i would expect linking against the imported lib should bring in this include directory as a transitive property, but also see that portaudio's CMakeLists.txt uses this BUILD_INTERFACE property that i'm not familiar with. it would be nice to get rid of this line because we could then move this section into `external_libraries/CMakeLists.txt` where it would be more appropriate.

Reply:
looking more into this, `PORTAUDIO_INCLUDE_DIRS` are set by `FindPortaudio`, if it is found. If this (and `PORTAUDIO_LIBRARIES`) are specified when running `FindPortaudio`, it assumes that PA has been found. Thus, we set these when building PA ourselves.